src/test/java/org/opensearch/security/privileges/PrivilegesEvaluatorUnitTest.java

```diff
@@ -8,14 +8,94 @@
 
 package org.opensearch.security.privileges;
 
+import com.google.common.collect.ImmutableList;
 import org.junit.Test;
 
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
-import static org.opensearch.security.privileges.PrivilegesEvaluator.isClusterPerm;
+import static org.opensearch.security.privileges.PrivilegesEvaluator.*;
```

Comment:
Can star import be removed here?

Reply:
OpenSearch refuses wildcard imports: https://github.com/opensearch-project/OpenSearch/blob/main/gradle/formatting.gradle#L89-L94

We should update the spotless formatting guidelines in this repo to also refuse wildcard imports

Reply:
Raised an issue in the repo for this: https://github.com/opensearch-project/security/issues/3260
Will create a follow up PR in main to fix the original violation and add the checkstyle. For now I just fixed it in this manual backport as well.